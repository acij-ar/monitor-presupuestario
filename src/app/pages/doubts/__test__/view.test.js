const React = require('react');
const { shallow } = require('enzyme');

const Doubts = require('../view');

describe('Doubts page', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Doubts pageName="doubts"/>);
    expect(wrapper).toMatchSnapshot();
  });
});
