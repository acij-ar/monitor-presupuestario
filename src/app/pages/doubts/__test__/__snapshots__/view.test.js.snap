// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Doubts page should match snapshot 1`] = `
<Page
  pageName="doubts"
>
  <div
    id="doubts-main-content"
  >
    <DoubtsMainContent />
    <Doubts />
  </div>
</Page>
`;
